From f05cd0ee20ebfb7ccf0cad5fa68b39fc2213a99e Mon Sep 17 00:00:00 2001
From: Prabhat Chand Pandey <prabhat.chand.pandey@intel.com>
Date: Thu, 4 Apr 2019 09:48:30 +0530
Subject: [PATCH] Device: Intel: Project-Celadon: Enable DBC RAW driver.

Enable DBC RAW driver as a built-in module.
At a time either DBC RAW or DBC TTY will work.

Tracked-On:
Signed-off-by: Prabhat Chand Pandey <prabhat.chand.pandey@intel.com>
---
 kernel_config/kernel_64_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/kernel_config/kernel_64_defconfig b/kernel_config/kernel_64_defconfig
index 9078eda..b51e580 100644
--- a/kernel_config/kernel_64_defconfig
+++ b/kernel_config/kernel_64_defconfig
@@ -4503,6 +4503,8 @@ CONFIG_USB_MON=y
 # CONFIG_USB_C67X00_HCD is not set
 CONFIG_USB_XHCI_HCD=y
 CONFIG_USB_XHCI_DBGCAP=y
+# CONFIG_USB_XHCI_DBGCAP_TTY is not set
+CONFIG_USB_XHCI_DBGCAP_RAW=y
 CONFIG_USB_XHCI_PCI=y
 CONFIG_USB_XHCI_PLATFORM=y
 CONFIG_USB_EHCI_HCD=y
--
2.21.0
